libs/sdk-ui-dashboard/src/topBar/DashboardTitle.tsx

```diff
@@ -11,19 +13,62 @@ export interface IDashboardTitleProps {
     title: string;
 
     /**
-     * Callback that the component MUST call when the title is clicked.
+     * Callback that the component calls when title edit enabled the title is changed. If not set, title
+     * taken as non-editable.
```

Comment:
Nitpick: taken as non-editable is a bit strange. What about "If no set, the title is considered non-editable"?

Reply:
But seeing there is isEditEnabled prop, maybe we should rephrase it. Something like 

> Callback that the component calls when title edit enabled the title is changed. Only relevant if isEditEnabled is set to true.

Reply:
The API here is not ideal, you can provide invalid combinations:
* `isEditable: true` and `onTitleChanged: undefined`
* `isEditable: false` and some `onTitleChanged`
Can we avoid it? Maybe by removing isEditable and using `onTitleChanged: undefined` as indicator that the item is not editable?